evaluator/src/main/kotlin/PackageRule.kt

```diff
@@ -104,11 +104,7 @@ open class PackageRule(
         val licenses = licenseView.licenses(pkg, detectedLicenses.map { it.license })
 
         licenses.forEach { (license, licenseSource) ->
-            val findings = if (licenseSource == LicenseSource.DETECTED) {
-                ruleSet.ortResult.collectLicenseFindings()[pkg.id].orEmpty()
-            } else {
-                emptyMap()
-            }
+            val findings = ruleSet.licenseFindings[pkg.id].orEmpty()
```

Comment:
Is dropping the condition intended here?

Reply:
Thanks, fixed.